src/utils/functions.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
File to remove : 
runFnWithState is used in ButtonRefresh but this button is to removed too.
isPromise is not used